runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -48,7 +48,7 @@ pub struct Config {
     /// Number of the transactions in the block.
     pub block_sizes: Vec<usize>,
     /// Where state dump is located in case we need to create a testbed.
-    pub state_dump_path: String,
+    pub state_dump_path: PathBuf,
```

Comment:
Why this is PathBuf instead of Path?

Reply:
`PathBuf` and `String` are owned containers (both backed by a `Vec<u8>` internally), `&Path` and `&str` are borrowed (pareametrized by a lifetime) counterparts. So this exchanges one owned container for another borrowed one with a more precise type.

Usning a borrowed container doesn't make much sense here: adding lifetime to a struct wound be a pain, and there isn't actually an existing owner we can borrow from here, so this won't even improve performance.

Reply:
@ailisp `PathBuf` is an owned version of `Path` (like `String` vs `str`)